css/styles.css

```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+/* Body general style */
+body {
+  display: flex;
+  flex-direction: column;
+  font-family: "Poppins", sans-serif;
+}
+
+/* Header */
+
+/* Navigation Bar */
+
+.navbar {
+  display: flex;
```

Comment:
- [ ]  Please use DRY code on your CSS, I can see a lot of repeated properties that can be wrapped in a class and used in the HTML instead of repeating multiple times in multiple classes. You can create a class : 
```
.d-flex{
   display: flex
}
```
And put d-flex class on HTML elements 👍🏻